Geometry/HGCalCommonData/interface/HGCalParameters.h

```diff
@@ -41,6 +41,12 @@ class HGCalParameters {
   hgtrform          getTrForm(unsigned int k) const;
   void              addTrForm(const CLHEP::Hep3Vector& h3v);
   void              scaleTrForm(double);
+  int               scintCells(const int layer) const
+  { return ((layer > layerFrontBH_[1]) ? nPhiBinBH_[1] : nPhiBinBH_[0]); }
```

Comment:
would it be possible to simplify to something like this (and similarly for the next function):
```
{ return nPhiBinBH_[scintType(layer)]; }
```
(It looks like the conditions should be consistent)